servers/common.go

```diff
@@ -282,6 +282,12 @@ type CRCProposalWithdrawInfo struct {
 	Signature      string `json:"signature"`
 }
 
+type CRDPOSManagementInfo struct {
+	CRManagementPublicKey string `json:"cr_managementpublickey"`
```

Comment:
crmanagementpublickey maybe is more proper

Reply:
Fixed it @Houshoupei84